browser/main/lib/ConfigManager.js

```diff
@@ -24,7 +24,8 @@ export const DEFAULT_CONFIG = {
   amaEnabled: true,
   hotkey: {
     toggleMain: OSX ? 'Command + Alt + L' : 'Super + Alt + E',
-    toggleMode: OSX ? 'Command + Option + M' : 'Ctrl + M'
+    toggleMode: OSX ? 'Command + Alt + M' : 'Ctrl + M',
+    deleteNote: OSX ? 'Command + Alt + Backspace' : 'Ctrl + Alt + Backspace'
```

Comment:
This hotkey is pure evil on Linux. It close all applications and go back to the login screen. Can you change it to `Ctrl + Shift + Backspace`

Reply:
Hahahaha yes